sessions/GraphQL-100/102/client/SOLUTION.md

```diff
@@ -0,0 +1,56 @@
+# SOLUTION
+
+## ProductList.js
+
+Fetch all products from the graphql server
+```
+  <Query
+    query={gql`
+      query { 
+        getProducts {
+  	      inStock, name, price, image, unitsAvailable, sku
+        }
+      }
+    `}
+  >
+```
+
+
+## ProductPage.js
+
+Fetch a product from the graphql server
+```
+const GET_PRODUCT = gql`
+  query($sku: Int!) { 
+    getProduct(sku:$sku) {
+      inStock, name, price, image, unitsAvailable, sku
+	  }
+  }
+`;
+```
+
+
+Create an Order
+```
+const CREATE_ORDER = gql`
+  mutation($sku: Int!) { 
+    createOrder(productSku: $sku) {
+      orderDate, product {name, inStock} 
+    }  
+  }
+`;
+```
+
+
+## OrdersPage.js
+
+Retrieve orders array
+```
+const GET_ORDERS = gql`
+  query {
+    getOrders {
+	    id, orderDate, product {name}
```

Comment:
Testing this out the orders list seem to be populated like this: `null` `product`, `null`, `product`
Which results in this error.
![image](https://user-images.githubusercontent.com/4874892/117172535-cfb67000-adc3-11eb-8ced-a48c28fb54f7.png)
When `Buying` a product the Id returned is also undefined. So perhaps related. 
![image](https://user-images.githubusercontent.com/4874892/117173930-27a1a680-adc5-11eb-8058-092c7ab24022.png)

Reply:
added missing `id` into solutions.md